.github/workflows/daily-vul-scan.yml

```diff
@@ -0,0 +1,72 @@
+name: daily vulnerability scan
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  IMAGE_NAME: zozo-gatling-operator
+  TRIVY_RESULTS_MARKDOWN: trivy-results.md
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  build:
```

Comment:
I feel the job should be separated more as not only building here.
Such as scanning and making an issue.

Reply:
As you suggested, it seemed like a good idea to separate the jobs.
However, upon consideration of the following points, I concluded that processing the trivy results within the same job would be more efficient than writing them out as a string and sharing across different jobs.
* The output of `aquasecurity/trivy-action@master` is specified by file.
* The input specification for `JasonEtco/create-an-issue@v2` is `filename`.

Consequently, I decided to only change the job name, instead of separating the jobs.

Reply:
OK, I understand it's tough to separate jobs.